tutordiscovery/patches/local-docker-compose-permissions-command

```diff
@@ -0,0 +1 @@
+{% if DISCOVERY_RUN_ELASTICSEARCH %}setowner 1000 /mounts/elasticsearch{% endif %}
```

Comment:
I am seeing `stat: can't stat '/mounts/elasticsearch': No such file or directory` error in permissions container. Can you verify this, please?